debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerTaskContext.java

```diff
@@ -15,7 +15,7 @@
  */
 public class SqlServerTaskContext extends CdcSourceTaskContext {
 
-    public SqlServerTaskContext(SqlServerConnectorConfig config) {
-        super("SQL Server", config.getLogicalName());
+    public SqlServerTaskContext(SqlServerConnectorConfig config, SqlServerDatabaseSchema schema) {
+        super("SQLServer", config.getLogicalName(), schema::tableIds);
```

Comment:
Is this the correct spelling? I find "SQL Server" (two words) more often on the web.

Reply:
@gunnarmorling I originally used it too but it is not practical when it is used in JMX names and dashboard queries

Reply:
Ok, I get the reasoning. I'd suggest to rename it to "SQL_Server" then. I'll do that change and apply.